src/Stratis.Bitcoin/P2P/PeerConnectorConnect.cs

```diff
@@ -69,8 +69,13 @@ public override void OnStartConnect()
                     return;
 
                 PeerAddress peerAddress = this.peerAddressManager.FindPeer(ipEndpoint);
-                if (peerAddress != null && !this.IsPeerConnected(peerAddress.Endpoint))
-                    await ConnectAsync(peerAddress).ConfigureAwait(false);
+                if (peerAddress == null)
```

Comment:
@thefazzer actually, can you completely revert the changes to this file? Then the PR is smaller 👍

Reply:
Not sure I understand the purpose?

Reply:
If I've done this correctly PeerConnectorConnect should now show no changes.